resources/views/listing/partials/filter/selected.blade.php

```diff
@@ -1,20 +1,32 @@
-<selected-filters :inner-class="{ button: '!hidden last:!inline-flex' }">
-    <x-rapidez::button
-        class="w-full mb-3 text-sm"
-        slot-scope="{ clearValues, selectedValues, components }"
-        v-on:click="clearValues"
+<selected-filters :inner-class="{ button: '!inline-flex' }">
```

Comment:
Is that prop / class still used now we're overwriting the slot?

Reply:
nope